opm/autodiff/StandardWell.hpp

```diff
@@ -302,11 +308,11 @@ namespace Opm
         void computeWellConnectionPressures(const Simulator& ebosSimulator,
                                                     const WellState& well_state);
 
-        // TODO: to check whether all the paramters are required
         void computePerfRate(const IntensiveQuantities& intQuants,
-                             const std::vector<EvalWell>& mob_perfcells_dense,
-                             const double Tw, const EvalWell& bhp, const double& cdp,
-                             const bool& allow_cf, std::vector<EvalWell>& cq_s,
+                             const std::vector<EvalWell>& mob,
+                             const EvalWell& bhp,
+                             const int perf,
+                             const bool allow_cf, std::vector<EvalWell>& cq_s,
```

Comment:
Nitpick: please put all of the arguments on separate lines, instead of only some (as was done in the _impl file).